packages/react-components/react-table/stories/DataGrid/Virtualization.stories.tsx

```diff
@@ -0,0 +1,195 @@
+};
+
+type AuthorCell = {
+  label: string;
+  status: PresenceBadgeStatus;
+};
+
+type Item = {
+  file: FileCell;
+  author: AuthorCell;
+  lastUpdated: LastUpdatedCell;
+  lastUpdate: LastUpdateCell;
+};
+
+const useScrollbarWidth = () => {
+  const { targetDocument } = useFluent();
+  return React.useMemo(() => {
+    if (!targetDocument) {
+      return 0;
+    }
+
+    const outer = document.createElement('div');
+    outer.style.visibility = 'hidden';
+    outer.style.overflow = 'scroll';
+
+    const inner = document.createElement('div');
+    outer.appendChild(inner);
+
+    targetDocument.body.appendChild(outer);
+    const scrollbarWidth = outer.offsetWidth - inner.offsetWidth;
+    outer.remove();
+    return scrollbarWidth;
+  }, [targetDocument]);
+};
+
+export const Virtualization = () => {
+  const scrollbarWidth = useScrollbarWidth();
+  const columns: ColumnDefinition<Item>[] = React.useMemo(
+    () => [
+      createColumn<Item>({
+        columnId: 'file',
+        compare: (a, b) => {
+          return a.file.label.localeCompare(b.file.label);
+        },
+        renderHeaderCell: () => {
+          return 'File';
+        },
+        renderCell: item => {
+          return <TableCellLayout media={item.file.icon}>{item.file.label}</TableCellLayout>;
+        },
+      }),
+      createColumn<Item>({
+        columnId: 'author',
+        compare: (a, b) => {
+          return a.author.label.localeCompare(b.author.label);
+        },
+        renderHeaderCell: () => {
+          return 'Author';
+        },
+        renderCell: item => {
+          return (
+            <TableCellLayout media={<Avatar badge={{ status: item.author.status }} />}>
+              {item.author.label}
+            </TableCellLayout>
+          );
+        },
+      }),
+      createColumn<Item>({
+        columnId: 'lastUpdated',
+        compare: (a, b) => {
+          return a.lastUpdated.timestamp - b.lastUpdated.timestamp;
+        },
+        renderHeaderCell: () => {
+          return 'Last updated';
+        },
+
+        renderCell: item => {
+          return item.lastUpdated.label;
+        },
+      }),
+      createColumn<Item>({
+        columnId: 'lastUpdate',
+        compare: (a, b) => {
+          return a.lastUpdate.label.localeCompare(b.lastUpdate.label);
+        },
+        renderHeaderCell: () => {
+          return 'Last update';
+        },
+        renderCell: item => {
+          return <TableCellLayout media={item.lastUpdate.icon}>{item.lastUpdate.label}</TableCellLayout>;
+        },
+      }),
+    ],
+    [],
+  );
+
+  const items = React.useMemo(() => {
+    const baseItems: Item[] = [
+      {
+        file: { label: 'Meeting notes', icon: <DocumentRegular /> },
+        author: { label: 'Max Mustermann', status: 'available' },
+        lastUpdated: { label: '7h ago', timestamp: 1 },
+        lastUpdate: {
+          label: 'You edited this',
+          icon: <EditRegular />,
+        },
+      },
```

Comment:
As we discussed in offline: this example needs some kind of growing index/id. As items are not random enough it's unclear if scroll happened during keyboard navigation.

Reply:
Updated the examples in https://github.com/microsoft/fluentui/pull/26081/commits/ab6489987d2014f2ecaf97351d70d06c608a3f5c